src/pages/index.js

```diff
@@ -110,9 +103,8 @@ function handleProfilePictureFormSubmit(url) {
       userInfo.setUserAvatar(data.avatar);
       changeProfilePicturePopup.close();
     })
-    .catch((err) => {
-      console.error(err);
-    })
+    .catch(console.error)
+
     .finally(() => {
       changeProfilePicturePopup.setLoading(false, "Save");
```

Comment:
You don't have `setLoading` in your project. Now it's called `renderLoading`. That's why there are errors when I try to submit a form